Cutelyst/utils.cpp

```diff
@@ -362,49 +362,46 @@ std::chrono::microseconds Utils::durationFromString(QStringView str, bool *ok)
             break;
         }
 
-        if (p.second == QLatin1StringView("usec") || p.second == QLatin1StringView("us")) {
+        if (p.second == QLatin1String("usec") || p.second == QLatin1String("us")) {
             ms += std::chrono::microseconds{dur};
-        } else if (p.second == QLatin1StringView("msec") || p.second == QLatin1StringView("ms")) {
+        } else if (p.second == QLatin1String("msec") || p.second == QLatin1String("ms")) {
```

Comment:
use p.second == u"msec"

Reply:
I wonder why they do not mention the char16_t overload in the docs. And isn’t it still more efficient to use QLatin1String(View) overload when available as mentioned in the QString docs?

Reply:
Yes, it's missing on the docs, but QString is a char16_t internally so using the u"" will cast to QStringView which can simply do a binary match instead of latin1 conversion, it's also mentioned that latin1 will be removed in Qt7 so I'm avoiding it.